fms/models/llama.py

```diff
@@ -379,7 +379,7 @@ def load_fms_llama(model_path: str, group=None):
     # IBM LLaMa
     fms_sd = _rename_weights_to_fms(checkpoint_sd)
 
-    extra_args = {}
+    extra_args = kwargs
```

Comment:
do we have a use-case for this?

Reply:
For example, passing `ntk_scaling=False` to the model init